nanostack-event-loop/platform/eventloop_config.h

```diff
@@ -19,8 +19,8 @@
 /*
  * Undefine all internal flags before evaluating the configuration
  */
-#undef NS_EVENTLOOP_USE_TICK_TIMER
-#undef NS_EXCLUDE_HIGHRES_TIMER
+//#undef NS_EVENTLOOP_USE_TICK_TIMER
```

Comment:
Aha! Never spotted the user config file. That's not the way any of the Nanomesh components have worked before, so wasn't expecting it.

(Usual convention is options can be passed in from Make, or picked up by known config systems like mbed or Yotta JSON).

Well, that works - I just think I'll expand the comments above the ifdefs to make it clearer.